locale/en/invitation.po

```diff
@@ -134,16 +134,10 @@ msgid "userInvitation.searchUser.nextButtonLabel"
 msgstr "Search User"
 
 msgid "userInvitation.searchUser.stepDescription"
-msgstr "Search for the user using their email address, username or ORCID ID. Enter at least one details to get started. If the user does not exist, you can invite them to take up roles and be a part of your journal. If the user already exist in the system, you can view user information and invite to take a additional roles."
+msgstr "If the user does not exist, you can invite them to take on roles and become part of your journal. If the user already exists in the system, you can view their information and invite them to take on additional roles."
```

Comment:
This references journals specifically, but shouldn't reference the specific context type since this is in pkp-lib.